packages/oui-select/src/select.directive.js

```diff
@@ -17,6 +17,7 @@ export default () => ({
         title: "@?",
         placeholder: "@?",
         items: "<",
+        disabledItems: "<",
```

Comment:
This property should be optional: `<?`
Don't forget to update the API table too :)